shared/constants/wallets.js

```diff
@@ -257,7 +257,7 @@ const paymentToCounterpartyType = (p: Types.Payment): Types.CounterpartyType =>
   return 'stellarPublicKey'
 }
 
-const paymentToYourRole = (p: Types.Payment, username: string): 'sender' | 'receiver' => {
+const paymentToYourRole = (p: Types.Payment): 'sender' | 'receiver' => {
```

Comment:
not sure is paymentToCounterpartyType (above) is entirely correct -- if source === target, then we won't have the name of the other account to display as the counterparty name. Will play around with it more...

Reply:
Played around with it more, and I'm getting inconsistent results when sending money from an account to itself. Sometimes `p.source === p.target === 'akalin'` (for my primary account) but sometimes `p.source` is the name of my secondary account and `p.target` is `akalin`. Will leave alone for now and file a bug with CORE...